recipes/tip/bld.bat

```diff
@@ -0,0 +1,22 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+cd build
+
+:: Hardcoded USER_VERSION is bad. Can't get previously un-defined
+:: env var that is defined in build.script_env to pass through. 
+:: How to do this?
+:: USER_VERSION is used to hardcode version in built executables.
+:: It is typically unnecessary because CMake configure-time uses
+:: git ops to discover tag or commit SHA, both of which are not 
+:: available since packages are built from tarballs.
+cmake .. -GNinja -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+    -DCONDA_PREFIX:PATH="%CONDA_PREFIX%" ^
+    -DUSER_VERSION:STRING=v1.0.2
```

Comment:
```suggestion
:: USER_VERSION is used to hardcode version in built executables.
:: It is typically unnecessary because CMake configure-time uses
:: git ops to discover tag or commit SHA, both of which are not 
:: available since packages are built from tarballs.
cmake .. -GNinja -DCMAKE_BUILD_TYPE:STRING=Release ^
    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
    -DCONDA_PREFIX:PATH="%CONDA_PREFIX%" ^
    -DUSER_VERSION:STRING="v%PKG_VERSION%"
```
Please check that the `v` is appropriate here for `USER_VERSION`. I think CMake versions also do not have a `v`.

https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process

Reply:
USER_VERSION is a legacy hard-coded version string for user convenience. The suggested change will work just fine.